src/js/step.js

```diff
@@ -345,22 +343,22 @@ export class Step extends Evented {
 
     this.tour.modal.setupForStep(this);
     this._styleTargetElementForStep(this);
-
     this.el.hidden = false;
 
-    this.tooltip.update();
```

Comment:
Are you sure we can remove this? Perhaps this was the wrong approach, but I think we needed to call `update` to ensure things got repositioned when scrolling or resizing the page etc.

Reply:
this seems to do extra renders, does the eventlistener for scroll call the initial show function?

Reply:
Unsure, but we need an update somewhere I think. Maybe I am wrong and popper now handles this internally?

Reply:
Does it update on scroll and resize without this?

Reply:
when it is out of viewport, there's a data attr added and we use that to set opacity to 0. What behavior where you expecting. Also, resize seems to have it's own internal events, as it does work as I'd expect.

Reply:
Just ensuring that the popper moves around and tries to stay in the viewport. I think on resize and scroll, the popper would stay in one spot before, if we weren't calling `update`.

Reply:
I checked this out locally, and it seems to be fine 👍